spec/const3.dd

```diff
@@ -370,7 +370,7 @@ struct S
     }
 }
 ---
-    $(P To make the return type $(D_KEYWORD immutable), you need to surround the return type with parentheses:
+    $(P To make the return type $(D_KEYWORD immutable), surround the return type with parentheses:
```

Comment:
The second use of "the return type" here is awkward. "surround it with" should be fine.